### PR TITLE
Task #14 chore: Improvement made in cluster package

### PR DESCRIPTION
**Pull request by pravinTek**

### CODE REVIEW

File: src/components/com_cluster/site/controllers/clusterusers.json.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+// No direct access
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Factory;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Component\ComponentHelper;
```
Comment (thite-amol) [inline src/components/com_cluster/site/controllers/clusterusers.json.php]: @pravinTek Remove unused import

File: src/components/com_cluster/site/controllers/clusterusers.json.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	 * Method to get user list depending on the client chosen.
+	 *
+	 * @return   null
+	 *
+	 * @since    1.0.0
+	 */
+	public function getUsersByClientId()
+	{
+		// Check for request forgeries.
+		Session::checkToken() or jexit(JText::_('JINVALID_TOKEN'));
```
Comment (thite-amol) [inline src/components/com_cluster/site/controllers/clusterusers.json.php]: @pravinTek Response using `JsonResponse`

File: src/components/com_cluster/site/controllers/clusterusers.json.php [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+		$clusterIds = Factory::getApplication()->input->getInt('cluster_id', 0);
+		$userOptions = $allUsers = array();
+
+		// Initialize array to store dropdown options
+		$userOptions[] = HTMLHelper::_('select.option', "", Text::_('COM_TJFIELDS_OWNERSHIP_USER'));
```
Comment (thite-amol) [inline src/components/com_cluster/site/controllers/clusterusers.json.php]: @pravinTek Language constant must be from cluster

File: src/components/com_cluster/site/controllers/clusterusers.json.php [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+		if (!empty($allUsers))
+		{
+			foreach ($allUsers as $user)
+			{
+				$userOptions[] = HTMLHelper::_('select.option', $user->user_id, trim($user->uname . ' (' . $user->uemail . ')'));
+			}
+		}
+
+		echo new JResponseJson($userOptions);
```
Comment (thite-amol) [inline src/components/com_cluster/site/controllers/clusterusers.json.php]: @pravinTek Use `JsonResponse`

**Review by thite-amol — CHANGES_REQUESTED** [review]

File: src/components/com_cluster/site/controllers/clusterusers.json.php (hunk 2 above)
Comment (pravinTek) [reply src/components/com_cluster/site/controllers/clusterusers.json.php]: Changed as per suggestion

File: src/components/com_cluster/site/controllers/clusterusers.json.php (hunk 3 above)
Comment (pravinTek) [reply src/components/com_cluster/site/controllers/clusterusers.json.php]: Used from com_cluster

File: src/components/com_cluster/site/controllers/clusterusers.json.php (hunk 4 above)
Comment (pravinTek) [reply src/components/com_cluster/site/controllers/clusterusers.json.php]: Used JsonResponse

**Pull request merged**